deepchem/molnet/defaults.py

```diff
@@ -9,47 +9,58 @@
 import json
 from typing import Dict, List
 
+from deepchem.feat.base_classes import Featurizer
+from deepchem.trans.transformers import Transformer
+from deepchem.splits.splitters import Splitter
+
 logger = logging.getLogger(__name__)
 
 
-def get_defaults(inspect_modules: bool = False) -> Dict[str, List[str]]:
+def get_defaults(module_name: str = None) -> Dict[str, object]:
```

Comment:
This new functions looks great! Could you add a unit test for it so we have a guard on its correctness?

Reply:
Instead of `object`, perhaps we should swap the type to read `Dict[str, Any]`?

Reply:
Definitely, I'll add some tests and swap to `Any`!